content/zh-cn/docs/reference/access-authn-authz/rbac.md

```diff
@@ -567,7 +567,7 @@ Here is an example that allows access to perform any current and future action o
 all current and future resources in the `example.com` API group.
 This is similar to the built-in `cluster-admin` role.
 -->
-你可愈使用通配符 `*` 可以批量引用所有的 `resources`、`apiGroups` 和 `verbs` 对象， 无需逐一引用。
+你可以使用通配符 `*` 可以批量引用所有的 `resources`、`apiGroups` 和 `verbs` 对象， 无需逐一引用。
```

Comment:
```suggestion
你可以使用通配符 `*` 批量引用所有的 `resources`、`apiGroups` 和 `verbs` 对象，无需逐一引用。
```
a nit

Reply:
I'm sorry. I just want to minimize the typos in the original text and forget to consider the fluency of the sentence. You're right.

Reply:
Please push again after your edit locally:

```
git add .
git commit --amend --no-edit
git push origin patch-1 -f
```